calico_rkt/calico_rkt.py

```diff
@@ -121,6 +110,12 @@ def __init__(self, network_config, env):
         to stdout at the end of execution.
         """
 
+        self.policy_driver = self._get_policy_driver()
+        """
+        Stores the policy driver based on the container runtime manager.
```

Comment:
I'd reword this, since eventually the policy driver might be chosen based on a number of things.  I'd say "Chooses the correct policy driver based on the given configuration"